Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 "2faa170" = {path = "./marbles/core", extras = ["dev"], editable = true}
 "55bbcad" = {path = "./marbles/mixins", extras = ["dev"], editable = true}
+pandas = "==0.23.1"
```

Comment:
I feel like we don't want this?

Reply:
How did this get in? Did you run something like `pipenv install pandas==0.23.1`?

Reply:
The `Pipfile` shouldn't reference packages that `marbles.mixins` depends on, generally. We should probably just delete this line, but I want to understand how it got in.

Reply:
I did indeed do that

Reply:
I'm not sure. I think the right thing to do is to update our `setup.py` files and then run `pipenv lock` to update the lockfile.